stdlib/domain.ml

```diff
@@ -129,56 +129,80 @@ module DLS = struct
 
 end
 
+(******** Identity **********)
+
+let get_id { domain; _ } = domain
+
+let self () = Raw.self ()
+
+let is_main_domain () = (self () :> int) == 0
```

Comment:
Nitpick: for me `== 0` is a code smell / C-ism, I would expect `= 0` instead.

Reply:
Fixed.